.travis.yml

```diff
@@ -7,6 +7,7 @@ before_install:
 install:
   - ./travis-tool.sh r_binary_install knitr
   - ./travis-tool.sh r_binary_install testthat
+  - ./travis-tool.sh r_install pryr
```

Comment:
Hmmmm, in that case I'd have a slight preference for inlining directly into magrittr. But that would make slightly harder to install from github, so using pryr is probably the best solution.